components/navigation/NavBar.js

```diff
@@ -32,7 +32,7 @@ export default function NavBar ({
   }, [open])
 
   return (
-    <div className={`bg-white ${className} z-50`}>
+    <div className={`bg-white ${className} z-50`} id="navbar">
```

Comment:
This would break reusability. Components should never have a fixed id because ids are unique per page.

Reply:
Well, that's how it is now. You never know in future designs :)

My proposal is that you make `id` a prop and pass it wherever it's used.

Reply:
Will it be fine if I can make this using className? In that case, you won't be having any issue with `id` in Components.